src/main/kotlin/no/skatteetaten/aurora/gobo/graphql/cname/CnameQuery.kt

```diff
@@ -0,0 +1,49 @@
+package no.skatteetaten.aurora.gobo.graphql.cname
+
+import com.expediagroup.graphql.generator.annotations.GraphQLIgnore
+import com.expediagroup.graphql.server.operations.Query
+import graphql.execution.DataFetcherResult
+import no.skatteetaten.aurora.gobo.graphql.newDataFetcherResult
+import no.skatteetaten.aurora.gobo.integration.gavel.CnameService
+import no.skatteetaten.aurora.gobo.integration.spotless.SpotlessCnameService
+import org.springframework.stereotype.Component
+
+@Component
+class CnameQuery(
+    val cnameService: CnameService,
+    val spotlessCnameService: SpotlessCnameService
+) : Query {
+
+    suspend fun cname(affiliations: List<String>? = null): Cname {
+        return Cname(affiliations, cnameService, spotlessCnameService)
+    }
+}
+
+data class Cname(
+    @GraphQLIgnore
+    val affiliations: List<String>?,
+    @GraphQLIgnore
+    val cnameService: CnameService,
+    @GraphQLIgnore
+    val spotlessCnameService: SpotlessCnameService
+) {
+    suspend fun azure(): DataFetcherResult<List<CnameAzure>> {
+        return try {
+            val cnameList = spotlessCnameService.getCnameContent()
```

Comment:
Spotless kunne egentlig fint hatt logikk for å ta imot affiliation som en queryparam slik at man slipper å returnere for hver eneste affiliation. 

Bør vi spille det inn som et behov? Det er tustelig at vi får tilbake så mye informasjon vi ikke er interessert i

Reply:
Her er Gavel og Spotless like i at de sender alt, så det burde kanskje spilles inn til begge?

Reply:
Dette er nå endret slik at Gobo sender affiliations som queryparam